packages/editor-ui/src/views/TemplatesSearchView.vue

```diff
@@ -132,6 +135,7 @@ export default defineComponent({
 			search: '',
 			searchEventToTrack: null as null | ISearchEvent,
 			errorLoadingWorkflows: false,
+			creatorHubUrl: CREATOR_HUB_URL as string,
```

Comment:
Is the type cast needed?

Reply:
Looks like it's not. Typescript was complaining it wasn't able to infer type for it but seems like it was just my editor acting weirdly